File: src/utils/formatters.ts
/**
 * Format currency in Colombian Pesos
 */
export const formatCurrency = (amount: number): string => {
  return `$${amount.toLocaleString('es-CO')}`;
};

/**
 * Format distance
 */
export const formatDistance = (distance: number): string => {
  if (distance < 1) {
    return `${Math.round(distance * 1000)} m`;
  }
  return `${distance.toFixed(1)} km`;
};

File: app/src/main/java/com/motodriver/app/ui/utils/Formatters.kt
package com.motodriver.app.ui.utils

import java.text.NumberFormat
import java.util.Locale

/**
 * Format currency in Colombian Pesos
 */
fun formatCurrency(amount: Int): String {
    val format = NumberFormat.getCurrencyInstance(Locale("es", "CO"))
    format.maximumFractionDigits = 0
    return format.format(amount)
}

/**
 * Format distance
 */
fun formatDistance(distance: Double): String {
    return if (distance < 1) {
        "${(distance * 1000).toInt()} m"
    } else {
        String.format(Locale.US, "%.1f km", distance)
    }
}
